portal-impl/src/com/liferay/portlet/announcements/service.xml

```diff
@@ -106,7 +106,7 @@
 		<!-- Audit fields -->
 
 		<column name="companyId" type="long" />
-		<column name="userId" type="long" />
+		<column name="userId" type="long" uad-anonymize-field-name="userId" />
```

Comment:
This is left on because this is the only uad attribute that can trigger UAD generation for AnnouncementsFlag